tests/integration_tests/test_integration.py

```diff
@@ -30,8 +30,16 @@
             [{"B01": 0.5, "B02": 0.75}, {"B01": 0, "B02": 1}, {"B01": -3, "B02": 4}, {"B01": None, "B02": None}],
             [1, 1.5, 0, 2, 0, 2, None, None],
         ),
-        ("test_count_with_condition", [{"B01": 0, "B02":1}, {"B01":2, "B02":3}, {"B01":4, "B02":5}, {"B01":None, "B02": None}], [1, 2]),
-        ("test_count_without_condition", [{"B01": 0, "B02":1}, {"B01":2, "B02":3}, {"B01":4, "B02":5}, {"B01":None, "B02": 3}], [3, 4]),
+        (
```

Comment:
Why was this changed?

Reply:
With the changes to `removeDimension` and consequently to `reduceByDimension` the latter function did not work correctly (I did not know that at the time and you pointed it out). In the meantime while I was unaware that it was not working correctly and I thought everything was ok, I changed the tests. Will change them back!

Reply:
Well that's why the tests are there 😄 If they fail, there should be something wrong with the code... Of course some time tests are wrong, but the first thought should be the code, especially if the output and input and the test setup are correct.